docs/how-to/migrate-from-pytest-operator.md

```diff
@@ -228,15 +231,15 @@ def test_active(juju: jubilant.Juju):
 It's common to use a `lambda` function to customize the callables further. For example, to wait specifically for `mysql` and `redis` to go active:
 
 ```python
-juju.wait(lambda status: jubilant.all_active(status, ['mysql', 'redis']))
+juju.wait(lambda status: jubilant.all_active(status, 'mysql', 'redis'))
```

Comment:
Would this be a good place to promote composition?

```py
juju.wait(lambda status:
    jubilant.all_active(status, 'mysql', 'redis') and
    jubilant.all_blocked(status, 'foobar'))
```

Reply:
Good idea -- done, and updated the text slightly to mention that